_includes/menu-item-ids/table-menu-items.md

```diff
@@ -25,3 +25,13 @@
 | `tablepasterowafter`      | Table                                                                                                    | Paste row after the currently selected row {{site.requires_5_4v}}          |
 | `tablerowprops`           | Table                                                                                                    | Opens the row properties dialog                                           |
 | `tabledeleterow`          | Table                                                                                                    | Deletes the currently selected row or rows {{site.requires_5_4v}}             |
+| `tableclass`              | _Not Applicable_                                                                                         | Adds pre-defined classes to selected table. {{site.requires_5_9v}}                                  |
```

Comment:
```suggestion
| `tabledeleterow`          | Table                                                                                                    | Deletes the currently selected row or rows {{site.requires_5_4v}}             |
| `tableclass`              | _Not Applicable_                                                                                         | Adds pre-defined classes to the selected table. {{site.requires_5_9v}}                                  |
```

Reply:
Also this button lets you remove classes too right?

Reply:
Yes.